src/hdusd/export/world/nodes.py

```diff
@@ -0,0 +1,66 @@
+#**********************************************************************
+# Copyright 2020 Advanced Micro Devices, Inc
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#********************************************************************
+from .node_parser import NodeParser
+
+
+class ShaderNodeOutputWorld(NodeParser):
+    def __init__(self, world, node, **kwargs):
+        super().__init__(world, node, None, **kwargs)
+
+    def export(self):
+        return self.get_input_link('Surface')
+
+
+class ShaderNodeBackground(NodeParser):
+    def export(self):
+        color = self.get_input_value('Color').data
+        strength = self.get_input_scalar('Strength').data
```

Comment:
redundant variables

Reply:
This done in such way to be compatible with code style in other nodes.